pyqir-generator/pyqir/generator/builder.pyi

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+class Builder:
+    """An instruction builder."""
+    ...
```

Comment:
Do any other attributes of `Builder` need to be listed here for type checking to succeed?

Reply:
No, for the initial version of the API there are no `Builder` attributes that are exposed in Python. They are internal to Rust only.